generators/entity-client/templates/react/src/test/javascript/e2e/entities/entity.spec.ts.ejs

```diff
@@ -194,11 +194,7 @@ describe('<%= entityClass %> e2e test', () => {
 
         expect(await isVisible(<%= entityInstance %>DeleteDialog.deleteModal)).to.be.false;
 
-        await waitUntilAnyDisplayed([<%= entityInstance %>ComponentsPage.noRecords,
-        <%= entityInstance %>ComponentsPage.table]);
-    
-        const afterCount = await isVisible(<%= entityInstance %>ComponentsPage.noRecords) ? 0 : await getRecordsCount(<%= entityInstance %>ComponentsPage.table);
-        expect(afterCount).to.eq(beforeRecordsCount);
+        await waitUntilDisplayed(<%= entityInstance %>ComponentsPage.noRecords);
```

Comment:
The React build for memcached failed because of this line.

Why only for memcached ? It's because:
- it's the only build with **dev** profile
- see https://github.com/jhipster/generator-jhipster/blob/master/.github/workflows/react.yml#L78
- see https://github.com/jhipster/generator-jhipster/blob/master/.github/workflows/react.yml#L102
- with dev profile, there are data, with fake-data

When I replaced with old code, it works well:
```
await waitUntilAnyDisplayed([<%= entityInstance %>ComponentsPage.noRecords, <%= entityInstance %>ComponentsPage.table]);
```

Reply:
Well spotted. This won't fix the issue, at least this shows there's room for improvement regarding the error messages, among other things.

Reply:
That is awesome @pascalgrimaud 

Let us move the `memcached` to `prod` build. Also we still are not fixing the root cause. The test cases should ideally be of the following format:

```
// setup data before each test case.
// do the action.
// validate the results.
```

Also, after discussing with @MathieuAA I strongly feel that we should move away from protractor to Cypress. This will solve a lot of problems. WDYT?

Reply:
About Cypress, it's another subject. FYI, Vue.js blueprint already started the work -> https://github.com/jhipster/jhipster-vuejs/pull/399

As Vue.js is not in main generator-jhipster, we could easily choose to remove Protractor or keep it, as we don't need to be totally consistent with main generator-jhipster. That's the power of blueprint.

Reply:
> > Let us move the memcached to prod build
> 
> It will be probably fixed the build. But don't forget a lot of us (in the core team) do demo, and start application with dev profile, and run e2e tests in dev profile. That's the reason I think e2e must work in dev profile too.
> 

Pascal, of course they must work no matter the env. The principal thing here is to be able to insert/remove data at will. That helps with the design issues we have with our tests. A solution could to have a third env, where data can be inserted/removed/whatever without messing with anything.

Reply:
> That's the reason I think e2e must work in dev profile too.

I do agree, it should work irrespective of the env. But this should be a temporary work around until we fix the test-case designs

> About Cypress, it's another subject. FYI, Vue.js blueprint already started the work

That is the beauty of Cypress, it will not tie you to a single framework. So, if we have built for vueJS then we can use it here.